column-profiling/common/src/main/java/com/r3/conclave/sample/dataanalysis/common/UserProfile.java

```diff
@@ -0,0 +1,32 @@
+package com.r3.conclave.sample.dataanalysis.common;
+
+public class UserProfile {
+    private String name;
+    private Integer age;
+    private String country;
+    private String gender;
+
+    public UserProfile(String name, Integer age, String country, String gender){
```

Comment:
Minor: Use Intellij's autoformat feature to format the code. The `{` should have a space before it, etc.

Reply:
Updated.